settings.gradle

```diff
@@ -1,3 +1,4 @@
 // Copyright 2004-present Facebook. All Rights Reserved.
 
-include ':ReactAndroid', ':Examples:UIExplorer:android:app', ':Examples:Movies:android:app', ':Examples:TicTacToe:android:app'
+include ':ReactAndroid', ':Examples:UIExplorer:android:app', ':Examples:Movies:android:app', ':Examples:TicTacToe:android:app', ':processor'
+project(':processor').projectDir = new File('ReactAndroid/src/main/third-party/java/processor')
```

Comment:
I guess this should be a top level module so you don't have to hack with project paths and sourceSets